prdoc/pr_4091.prdoc

```diff
@@ -0,0 +1,14 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Remove `pallet::getter` usage from authority-discovery pallet
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removes `pallet::getter`s from `pallet-authority-discovery`.
+      The syntax `StorageItem::<T, I>::get()` should be used instead.
+
+crates:
+  - name: pallet-authority-discovery
+    bump: minor
```

Comment:
this is a breaking change
```suggestion
    bump: major
```

Reply:
Updated 👍 Though the storage values are private, no? Or is it that someone might have customized the pallet and this could break their version?

Reply:
The pallet exposes the getters in a public api, so this breaks that public api